charts/argo-cd/templates/redis/deployment.yaml

```diff
@@ -6,7 +6,6 @@ metadata:
   name: {{ template "argo-cd.redis.fullname" . }}
   labels:
     {{- include "argo-cd.labels" (dict "context" . "component" .Values.redis.name "name" .Values.redis.name) | nindent 4 }}
-    app.kubernetes.io/version: {{ .Values.redis.image.tag | quote }}
```

Comment:
Please restore the version label change. Redis uses its own version which does not belong to the chart version

Reply:
done.